index.html

```diff
@@ -0,0 +1,169 @@
+        <h4 class="headline__location">@ Sedona Red Rocks, USA</h4>
+      </div>
+    </section>
+    <section id="features">
+      <div class="flex-container">
+        <div class="features__head">
+          <h2>Activities</h2>
+          <hr class="section__hr" />
+        </div>
+        <div class="features__grid">
+          <div class="features__cards">
+            <i class="fa fa-plane-departure"></i>
+            <h3>Sky Diving</h3>
+            <p>
+              Skydiving is to free-fall until a certain altitude from an
+              aircraft or a mountain, fly about 30 to 180 seconds, and then
+              float in the air until you land on safe ground with a parachute.
+            </p>
+          </div>
+          <div class="features__cards">
+            <i class="fa fa-parachute-box"></i>
+            <h3>Para Gliding</h3>
+            <p>
+              Paragliding is flying in the air using paragliders. These
+              paragliders are light in weight and are foot launched. There is a
+              harness in the glider on which the paraglider sits.
+            </p>
+          </div>
+          <div class="features__cards">
+            <i class="fa fa-tachometer-alt"></i>
+            <h3>Para Motoring</h3>
+            <p>
+              Paramotoring is a form of ultralight aviation where the pilot
+              wears a back-pack motor (a paramotor) which provides enough thrust
+              to take off using a paraglider.
+            </p>
+          </div>
+          <div class="features__cards">
+            <i class="fa far fa-paper-plane"></i>
+            <h3>Hang Gliding</h3>
+            <p>
+              Hang Gliding is a pilot flies a light, non-motorised aircraft
+              called a hang glider. Most modern hang gliders are made of an
+              aluminium alloy frame covered with synthetic sailcloth to form a
+              wing.
+            </p>
+          </div>
+          <div class="features__cards">
+            <i class="fa fa-plane"></i>
+            <h3>Wingsuit Flying</h3>
+            <p>
+              Wingsuit Flying is wearing a webbing-sleeved jumpsuit called a
+              wingsuit to add wetted area to the diver's body and generate
+              increased lift, which allows extended air time by gliding flight.
+            </p>
+          </div>
+        </div>
+        <a class="participate-link" href="">SEE THE CATALOGUE</a>
+        <button id="participate">Dive With Us!</button>
+      </div>
+    </section>
+    <section id="staff">
+      <div class="flex-container">
+        <div class="staff__head">
+          <h2>Our Instructors</h2>
+          <hr class="section__hr" />
+        </div>
+      </div>
+    </section>
+    <section id="footer-main">
+      <div class="flex-container">
+        <img src="assets/images/air-sports-logo.jpg" alt="Air Sports logo" />
+        <p>Copyright ©️ 2022 Air Sports Club. Some Rights Reserved</p>
+      </div>
+    </section>
```

Comment:
Also please note that the footer should have a white background on the mobile version as shown in the template 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/190353111-e6b5166e-687a-4cfb-90e0-f0940af91582.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/190353249-5932ac15-1c8e-4286-9a9e-2f18c2b1f0cd.png)